plugins/scaffolder-common/src/ScaffolderClient.ts

```diff
@@ -329,6 +330,16 @@ export class ScaffolderClient implements ScaffolderApi {
 
           const logs = (await response.json()) as LogEvent[];
 
+          logs.forEach(log => {
```

Comment:
It can be done without an extra loop, and `createdAt` is not optional field in LogEvent.

```
const toLocalTimeZone = (isoTime: string) =>  DateTime.fromISO(isoTime, {
                zone: 'utc',
              })
                .setZone(DateTime.local().zoneName)
                .toISO() as string;

for (const event of logs) {
   event.createdAt  = toLocalTimeZone(event.createdAt);
   ....
}
```